apps/meteor/client/sidebar/sections/components/OmnichannelMakeCall.tsx

```diff
@@ -0,0 +1,31 @@
+import { Sidebar } from '@rocket.chat/fuselage';
+import { useMutableCallback } from '@rocket.chat/fuselage-hooks';
+import React, { ReactElement } from 'react';
+
+import { useOutboundDialer } from '../../../../ee/client/hooks/useOutboundDialer';
+
+export const OmnichannelMakeCall = (): ReactElement => {
```

Comment:
This component needs to be removed. It has already been removed in the original branch.